### PR TITLE
dev-libs/libnv-portable: new package

### PR DESCRIPTION
**Pull request by Zer0-One**
Bug: https://bugs.gentoo.org/730350
Signed-off-by: David Zero <zero-one@zer0-one.net>

### CODE REVIEW

File: dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+EAPI=7
+
+inherit multibuild cmake
+
+DESCRIPTION="A userspace version of libnv ported from FreeBSD"
+HOMEPAGE="https://github.com/Zer0-One/libnv-portable"
+SRC_URI="https://github.com/Zer0-One/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
```
Comment (chutz) [inline dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild:13]: Given this is a C library, it probably should have a subslot. Generally the version on the default soname works (`0/1`) is generally a reasonable option. This allows one to increment this if upstream breaks ABI.

File: dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+KEYWORDS="amd64"
```
Comment (chutz) [inline dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild:14]: Generally new ebuils should be `~amd64`

File: dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+IUSE="static-libs test"
+
+DEPEND="${RDEPEND} test? ( dev-util/kyua dev-libs/atf )"
```
Comment (chutz) [inline dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild:17]: This should be expanded for readability (and nicer diffs when comparing versions)
```ebuild
DEPEND="${RDEPEND}
 	test? (
 	 	dev-util/kyua
 	 	dev-libs/atf
 	)
"

File: dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+RDEPEND=">=dev-libs/libbsd-0.10.0[static-libs?]"
```
Comment (chutz) [inline dev-libs/libnv-portable/libnv-portable-1.0.1.ebuild:18]: Generally C library dependencies should have a subslot operator
```ebuild
RDEPEND=">=dev-libs/libbsd-0.10.0:=[static-libs?]"
```

**Comment by ajakk** [comment]
Thanks for working on this! If still interested, I think [GURU](https://wiki.gentoo.org/wiki/Project:GURU) is the best place for you to add it.

**Pull request closed without merging**